shared/js/background/safari-events.es6.js

```diff
@@ -158,7 +157,7 @@ let onBeforeRequest = (requestData) => {
 
     let redirectRequest = redirect.handleRequest(requestData)
     if (redirectRequest) {
-        redirectRequest
+        return redirectRequest
```

Comment:
cc @jdorweiler - not sure if this was a bug or not. It seems like we would've noticed it if trackers weren't blocked properly on Safari.

Reply:
Yeah still works but it should have a return.